### PR TITLE
Soft 233 controller board aliases

### PR DESCRIPTION
**Pull request by AveryChiu64**

### CODE REVIEW

**Comment by copperium** [comment]
Looks like Travis is complaining about the formatting, it'll need a `make format`.

**Comment by AveryChiu64** [comment]
> Looks like Travis is complaining about the formatting, it'll need a `make format`.

Hmm that's strange, my hooks run make format automatically and it formats it in this strange way.

File: libraries/ms-helper/inc/controller_board_pins.h [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+// Defined as a GPIO Address
+
+// CAN
+#define CONTROLLER_BOARD_ADDR_DEBUG_CAN_RX \
+                                           \
+  { GPIO_PORT_A, 11 }
+
+#define CONTROLLER_BOARD_ADDR_DEBUG_CAN_TX \
+                                           \
+  { GPIO_PORT_A, 12 }
```
Comment (copperium) [inline libraries/ms-helper/inc/controller_board_pins.h:12]: Why debug CAN rx/tx? isn't it just CAN rx/tx?

File: libraries/ms-helper/inc/controller_board_pins.h [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                                              \
+  { GPIO_PORT_B, 6 }
+
+#define CONTROLLER_BOARD_ADDR_DEBUG_USART1_RX \
+                                              \
+  { GPIO_PORT_B, 7 }
+
+// USART 2
+#define CONTROLLER_BOARD_ADDR_USART2_TX \
+                                        \
```
Comment (copperium) [inline libraries/ms-helper/inc/controller_board_pins.h:25]: nit: not sure why the extra blank lines between the definition and the pin, were those added by `make format`? they should be removed

**Review by copperium — CHANGES_REQUESTED** [review]
2 tiny nits, looks good

File: libraries/ms-helper/inc/controller_board_pins.h (hunk 1 above)
Comment (AveryChiu64) [reply libraries/ms-helper/inc/controller_board_pins.h:12]: image

**Review by copperium — APPROVED** [review]

**Pull request merged**